frontends/web/src/components/transactions/transaction.jsx

```diff
@@ -208,42 +140,7 @@ export default class Transaction extends Component {
                         </div>
                         <div className={[parentStyle.action, parentStyle.showOnMedium].join(' ')}>
                             <a href="#" className={style.action} onClick={this.showDetails}>
-                                {
-                                    !transactionDialog ? (
```

Comment:
why is there two times code that looks almost identical?

Reply:
The reason for the 2 different "show details" button is the different positioning for the responsive design.  

BTW: I'll get rid of `<a href="#" ` and replace it by button or just add click handler on the arrow.

![Screen Shot 2020-06-17 at 1 28 04 PM](https://user-images.githubusercontent.com/546900/84907614-b5907800-b0b3-11ea-8f67-831d814eb18f.png)
![Screen Shot 2020-06-17 at 1 36 31 PM](https://user-images.githubusercontent.com/546900/84907624-b88b6880-b0b3-11ea-8184-54a5731e2280.png)

Reply:
I'd like to leave it for now and merge this PR soon™